lib/lynis/lynistest.pm

```diff
@@ -51,7 +51,8 @@ our $testdir = "/tmp/";
 our $f_position_b = 0;
 our $f_position_c = 0;
 
-our $lynis_baseline_file_default = "baseline-lynis-audit-system-nocolors-sle15sp3-x86_64-snapshot7-textmode";
+my $var_str = get_var("VERSION", "15-SP3") . "-" . get_var("ARCH", "x86_64") . "-" . get_var("DESKTOP", "textmode");
```

Comment:
Is it necessary to hardcode a specific SLE version here and in the files attached? I assume this will cause a lot of maintenance work needing recurring updates for every service pack.

Also, what about a corresponding TW test? Because of [Factory First](https://opensource.suse.com/suse-open-source-policy) we should test against Tumbleweed by default (with exceptions for special packages, of course). Could you please check if it works with a current TW snapshot and then add to the main.pm there?

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
> Is it necessary to hardcode a specific SLE version here and in the files attached? I assume this will cause a lot of maintenance work needing recurring updates for every service pack.
> 
Good question, The reason is there are might some but unimportant differences in the outputs (or system testing results) of `lynis audit system` between different products (just like between different architectures), and the outputs analyzing is really hard as the outputs is complicated/changeable So I prefer to make the baseline more accurate other than using a general one otherwise I have to write complicated code according to changeable outputs.

> Also, what about a corresponding TW test? Because of [Factory First](https://opensource.suse.com/suse-open-source-policy) we should test against Tumbleweed by default (with exceptions for special packages, of course). Could you please check if it works with a current TW snapshot and then add to the main.pm there?
> 
Sure, I will.
> See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details